tests/tracing/test_pruning.py

```diff
@@ -1280,13 +1279,13 @@ def test_prune_by_length(
                 ],
                 dtype=np.uint8,
             ),
-            id="Height pruning based on median, height threshold 8.0e-19",
+            id="Height pruning based on median, height threshold 7.3",
             # marks=pytest.mark.skip(),
         ),
         pytest.param(
             "pruning_skeleton",
             None,
-            7.7e-19,
+            7.1,
```

Comment:
Ditto, value and magnitude have changed?

Reply:
Similar story for this other one that changed.